docs/team/hansstanley.md

```diff
@@ -5,36 +5,50 @@ title: Stanley Han's Project Portfolio Page
 
 * **Project management**:
   * Initialised team organisation and tP repository
   * Created PR to the module's `master` branch
   * Enabled issue tracker and GitHub Actions (for CI and codecov)
-  * Set up project website
+  * Set up and adapted the project website for NeoBook
+  * Triaged issues from PE-D
 
 * **Enhancements to existing features**:
   * Redesigned the GUI
-    * Changed the layout (PRs [#39](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/39), [#56](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/56), [#57](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/57), [#59](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/59), [#62](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/62), [#74](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/74), [#87](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/87), [#96](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/96), [#107](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/107), [#131](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/131))
-    * Added light and dark themes (PR [#129](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/129))
+    * Changed the layout (PRs [#39](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/39), [#56](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/56), [#57](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/57), [#59](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/59), [#62](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/62), [#74](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/74), [#87](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/87), [#96](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/96), [#107](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/107), [#131](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/131)).
+    * Split the single contacts list panel into list and details panels to accommodate the increased number of information fields without cluttering the list.
+    * Moved the command box and result display to the bottom of the window, borrowing familiarity from popular desktop messaging applications.
+    * Added a tabbed view for the events and "me" panels to avoid cluttering the screen space, while borrowing familiarity from the widely used Outlook desktop client.
+    * Added a system of CSS styling options to increase code reuse.
+      * Labels have a series of font sizes (`h1..h6, p`).
+      * Paddings have predefined styling (e.g., `pt` for top padding, `pa-l` for "all except left" padding).
+      * The backgrounds for labels and panes have 3 colour types: `normal`, `accented`, and `muted`. Generally, `muted` is used for background components, while `accented` is used in components containing information useful to the user.
+  * Enhanced the help window with the list of all commands (PR [#181](https://github.com/AY2223S2-CS2103T-F12-3/tp/pull/181)).
+    * The list uses the `MESSAGE_USAGE` from each of the commands, making it consistent with the help messages that show up in the result display.
 
```

Comment:
I think you can add bug fixes PRs as well since you did fix quite a few of the,m